### PR TITLE
added contributing.rst

### PR DESCRIPTION
**Pull request by oke-aditya**
Added contributing.rst file. To enable contributors.

This file followed the steps as listed in contributing.rst (bit recursive).

### CODE REVIEW

File: contributing.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+.. code:: bash
+    
+    $ git clone https://github.com/username/mantisshrimp.git
+
+3. Install the requirments. You many use miniconda or conda as well.
+
+.. code:: bash
+    
+    $ pip install -r requirments.txt
```
Comment (lgvaz) [inline contributing.rst:31]:
```suggestion
    $ pip install -r requirements.txt
```

**Comment by lgvaz** [comment]
Very nice!!

Can you add a link to the contributing guide at the readme?

File: contributing.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+1. After you update the repo from mantisshrimp/master, create a new branch from the local master branch.
+
+.. code:: bash
+
+   $ git checkout -b feature-name
+   $ git branch
+    master 
+    * feature_name
+
+2. Make changes. Edit files in your favorite editor and format the code with black.
```
Comment (lgvaz) [inline contributing.rst:60]: Can you add a link to black github repo?

**Comment by lgvaz** [comment]
Also, everything until "Follow these steps" can be removed

**Comment by oke-aditya** [comment]
Okay I will do that.

**Comment by lgvaz** [comment]
Thanks for the PR!

**Pull request merged**